config/seo-pro.php

```diff
@@ -34,4 +34,7 @@
         'card' => 'summary_large_image',
     ],
 
+    'alternate_locales' => [
+        'enabled_in_meta' => true,
+    ],
```

Comment:
This is a great PR 👍

We had a chat about it though, and would ask that you refactor the config from this:

```php
'alternate_locales' => [
    'enabled_in_meta' => true,
],
```

To this:

```php
'alternate_locales' => true,
```